draft-ietf-quic-transport.md

```diff
@@ -4085,6 +4085,13 @@ UDP datagrams MUST NOT be fragmented at the IP layer.  In IPv4
 ({{!IPv4=RFC0791}}), the DF bit MUST be set if possible, to prevent
 fragmentation on the path.
 
+Even though datagrams with certain properties are required to be padded, the
+size of the datagram is not authenticated, and endpoints might send coalesced
+packets after the handshake is confirmed (see {{packet-coalesce}}).  Therefore,
```

Comment:
Once the handshake is confirmed, Handshake keys are dropped. From that point on, it's not possible to send coalesced packets any more.

Reply:
> Once the handshake is confirmed, Handshake keys are dropped. From that point on, it's not possible to send coalesced packets any more.

That's not true. Use of handshake packets is explicitly allowed post-handshake; see
https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#name-pmtu-probes-containing-sour.

The reason the reference currently points to packet-coalesce is because it's more generic (and also contains reference to post-handshake PMTUD), but the reference can be specific if that's others' preference.